README.md

```
@@ -90,6 +90,10 @@ TODO: `obligatory_fields` is a new subsection, start using it in the actual appl
 ```
 "accessible_by": ["bikes", "cars", "pedestrians"]
 ```
+- `categories_default_checked` - for a given category, the options that should be pre-checked in the filter panel when the app first loads. E.g.
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)